app/templates/admin/sessions/index.hbs

```diff
@@ -0,0 +1,56 @@
+<div class="sixteen wide column">
```

Comment:
Remove this file and make the index route use `admin/sessions/list.hbs` template.